client/src/components/pages/Login.tsx

```diff
@@ -1,39 +1,30 @@
-import {
-  FormControl,
-  FormLabel,
-  Input,
-  Button,
-  Heading,
-  Box,
-  VStack
-} 
-from "@chakra-ui/react";
+import { FormControl, FormLabel, Input, Button, Heading, Box, VStack } from "@chakra-ui/react";
 import { useState } from "react";
-import { UserCredientials } from "../../types";
+import { UserCredentials } from "../../types";
 import { userCredentialsSchema } from "../../schemas";
 
 const Login = () => {
-  const [loginBody, setLoginBody] = useState<UserCredientials>({
+  const [loginBody, setLoginBody] = useState<UserCredentials>({
     username: "",
-    password: ""
+    password: "",
   });
 
   const handleLogin = () => {
-    const parsedLoginBody = userCredentialsSchema.safeParse(UserCredientials);
+    const parsedLoginBody = userCredentialsSchema.safeParse(loginBody);
     if (!parsedLoginBody.success) {
       alert("Login import failed!");
       return;
     }
-    if (loginBody.username == '' || loginBody.password == '') {
+    if (loginBody.username == "" || loginBody.password == "") {
       alert("Please enter a username and password!");
       return;
     }
     alert(JSON.stringify(parsedLoginBody.data));
-  }
+  };
 
   return (
-    <Box bg="#156087">
-      <VStack maxW="2xl" textColor="white" padding="3rem" gap="4" marginInline="auto">
+    <Box bg="#156087" display="flex" minH="calc(100vh)">
```

Comment:
I'm pretty sure ChakraUI has a property that means `100vh`, so you can use that instead of a `calc` workaround. I think it's called something like `full` or `screen`. We are using TypeScript so if you just start typing it will show all possible values for `minH`,

Reply:
I know this is a nitpick, but I just think its better to use what the library intends.